beginner/apps/src/bin/radio-recv.rs

```diff
@@ -37,14 +38,15 @@ fn main() -> ! {
 
     match res {
         Ok(crc) => {
-            log::info!(
-                "received: {} (CRC={})",
+            defmt::info!(
+                "received: {} (CRC = {:X})",
+                //                    ^^ print as uppercase hexadecimal
                 str::from_utf8(&*packet).expect("response is not valid UTF-8"),
                 crc
             );
         }
-        Err(Error::Crc(crc)) => log::error!("invalid CRC: {:06x}", crc),
-        Err(Error::Timeout) => log::error!("no response within {} ms", TEN_MS / 1_000),
+        Err(Error::Crc(crc)) => defmt::error!("invalid CRC: {=u16:x}", crc),
```

Comment:
is the type annotation (`=u16`) required here? I would prefer to avoid syntax that differs from `println!` to not have "yet another thing that needs to be explained"

Reply:
good point, fixed